self-hosted-login/okta-aspnet-mvc-example/Views/Account/Login.cshtml

```diff
@@ -0,0 +1,30 @@
+﻿@{
+    ViewData["Title"] = "Login";
+}
+
+<!-- Latest CDN production Javascript and CSS: 2.6.0 -->
+<script src="https://ok1static.oktacdn.com/assets/js/sdk/okta-signin-widget/2.6.0/js/okta-sign-in.min.js" type="text/javascript"></script>
```

Comment:
FYI, this documentation is old :(
Latest production version is 2.12, soon to be 2.13: https://github.com/okta/okta-signin-widget/releases